toolset/benchmark/fortune_html_parser.py

```diff
@@ -25,140 +26,146 @@ class FortuneHTMLParser(HTMLParser):
 <tr><td>12</td><td>フレームワークのベンチマーク</td></tr>
 </table></body></html>'''
 
-  # Is called when a doctype or other such tag is read in.
-  # For our purposes, we assume this is only going to be
-  # "DOCTYPE html", so we will surround it with "<!" and ">".
-  def handle_decl(self, decl):
-    # The spec says that for HTML this is case insensitive,
-    # and since we did not specify xml compliance (where
-    # incorrect casing would throw a syntax error), we must
-    # allow all casings. We will lower for our normalization.
-    self.body.append("<!{d}>".format(d=decl.lower()))
+    # Is called when a doctype or other such tag is read in.
+    # For our purposes, we assume this is only going to be
+    # "DOCTYPE html", so we will surround it with "<!" and ">".
+    def handle_decl(self, decl):
+        # The spec says that for HTML this is case insensitive,
```

Comment:
So this should be a `'''` comment block?

Reply:
Prolly